src/themes/teams/components/Icon/svg/icons/index.ts

```diff
@@ -0,0 +1,17 @@
+export { default as call } from './call'
+export { default as callEnd } from './callEnd'
+
+export { default as callVideo } from './callIncomingVideo'
```

Comment:
the reason was to save mapping from the icon that is provided (with simple name) to the one that was originally used by Teams (longer name) - with that I thought it will be easier to track which one corresponds to which at the initial stages of adoption. Absolutely ok to make any renaming that will suit Teams theme devs